ga4gh/datamodel/variants.py

```diff
@@ -903,7 +917,7 @@ def convertLocationHgvsP(self, hgvsp):
 
     def addCDSLocation(self, effect, cdnaPos):
         hgvsC = effect.hgvsAnnotation.transcript
-        if hgvsC != '':
+        if hgvsC != '' and hgvsC is not None:
```

Comment:
Personally, I have also found that treating non-boolean types as
implicit booleans is unclear.

A function that obviously documents intention over
representation is clearer.  Assuming this is testing for
something being unspecified a function isUnspecified(value)
communicates more than either `if hgvsC ' or
`if hgvsC == "" or hgvsC is None' and is less verbose than the
latter.

David Steinberg <notifications@github.com> writes:
> Python provides the idiom if hgvsC which checks both cases, but I changed it
> elsewhere at your request because it wasn't verbose enough.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>